### ISSUE TITLE
Hourly forecast inconsistent and generally cramped

### ISSUE THREAD

**Issue by Djtpj** [atticuscornett/AtmosWeather#2]
The hourly forecast for the Android deployment is highly inconsistent, and feels extremely cramped, or it is not adequately taking up the allotted space.

Below I have attached and excellent example of this.
![Screenshot_20221023-105127_Atmos Weather.jpg](https://user-images.githubusercontent.com/93952263/197402353-c6735da7-c679-470a-ad4d-c21605d6a3e0.jpg)

It is not clear to me why you cannot use a flexbox with justify space? Or better yet, a grid with three columns set to 1fr each. However these approaches might requre a media query.

**Comment by atticuscornett** [comment]
Hey @Djtpj! Thanks for leaving a suggestion. Fixing this has been in our development backlog for a while, but I moved it to high priority. (You can find the development Notion board in the development section of the README.) I likely won't be fixing this or making any modifications to Atmos Weather until the Congressional App Challenge judging period is over.

**Comment by atticuscornett** [comment]
Hello again, @Djtpj! Sorry for the radio silence over the past couple of months. The Congressional App Challenge judging is complete, and that means I can work on this again! As promised, I made this a top priority when getting back to work. As far as I have been able to tell in emulation, this issue has been resolved by the latest commit (6bdce75). After I have total confirmation that this has solved the issue, I will mark this issue as closed.
